src/server.rs

```diff
@@ -102,23 +102,23 @@ impl Server {
         asyncio
             .call_method1("set_event_loop", (event_loop,))
             .unwrap();
-        let event_loop_hdl = Arc::new(PyObject::from(event_loop));
-        let event_loop_cleanup = event_loop_hdl.clone();
         let startup_handler = self.startup_handler.clone();
         let shutdown_handler = self.shutdown_handler.clone();
 
+        let task_locals = Arc::new(pyo3_asyncio::TaskLocals::new(&event_loop).copy_context(py)?);
+        let task_locals_cleanup = task_locals.clone();
```

Comment:
as PyO3 update there APIs to use task_locals instead of `event_loop`.

Reply:
@suhailmalik07 , this looks great thanks. Can you just point me to the `task_locals` documentation too?

Reply:
Sure, In documentation. It's mentioned under solution.
Link: https://docs.rs/pyo3-asyncio/0.17.0/pyo3_asyncio/#the-solution

Here in migration, It's also mentioned.

<img width="641" alt="image" src="https://user-images.githubusercontent.com/40052045/198701302-34aba52d-58ff-4635-b557-1915d2db5aa0.png">

Link -> https://crates.io/crates/pyo3-asyncio